app/(main)/markets/AiChat.tsx

```diff
@@ -250,4 +290,17 @@ const LoadingDots = () => {
   );
 };
 
+interface CardProps extends PropsWithChildren {
+  title: string;
+}
+
+const Card = ({ title, children }: CardProps) => {
+  return (
+    <div className="min-w-40 space-y-1 rounded-20 bg-outline-primary-base-em p-4">
+      <p className="text-sm font-semibold uppercase text-text-low-em">{title}</p>
+      <p className="text-xl font-bold uppercase">{children}</p>
```

Comment:
Improve the styles a bit, as the card already enhances the numbers, we don't need it as big and bold. IMO like this works better.
```suggestion
      <p className="text-md font-semibold uppercase">{children}</p>
```

<img width="456" alt="image" src="https://github.com/user-attachments/assets/d417e24a-9eb6-4621-9480-5c40c8c627bf" />